pkg/pipeline/decode/decode_protobuf.go

```diff
@@ -1,3 +1,20 @@
+/*
+ * Copyright (C) 2022 IBM, Inc.
```

Comment:
Do we have to distinguish between files created and copyright by IBM vs Redhat?

Reply:
I don't know
cc @eranra @jotak @jpinsonneau @OlivierCazade @mariomac